blueprints/test-framework-detector.js

```diff
@@ -11,14 +11,22 @@ module.exports = function(blueprint) {
     var type;
 
     var dependencies = this.project.dependencies();
-    if ('ember-qunit' in dependencies || 'ember-cli-qunit' in dependencies) {
+    if ('ember-qunit' in dependencies) {
       type = 'qunit';
```

Comment:
I think this can actually be tweaked to assume "new style" can't it?  Basically, there are no known (to me at least) versions of ember-qunit taht supported usage as an addon but did _not_ support the RFC232 style...

Reply:
whoops, that was supposed to be the case, but I forgot to change it...

Reply:
fixed